doc(release): Release notes for map 23.04.18 and mbi 23.04.10 (#3452)
centreon/centreon-documentation: versioned_docs/version-23.04/releases/centreon-commercial-extensions.mdx

## Centreon MAP

### 23.04.18

Release date: `July 29, 2024`

<details open>
  <summary>Enhancements</summary>

> - [Selinux] Packaged selinux rules for centreon-map and centreon-mbi.

Reviewer: ```suggestion
- [Selinux] Packaged SELinux rules.
```

</details>

<details>
  <summary>Bug fixes</summary>

- [ACL] Fixed an issue which caused some users to be able to duplicate maps when they shouldn't.
- [Diagnostic] Fixed an issue with the diagnostics tool that occurred when MAP was configured in HTTPS.
- [Editor] Fixed an issue causing link labels not to be displayed in editor mode.
- [Map list] Fixed an issue that could prevent the list of maps from being displayed.
- [Server] Fixed an issue with HTML tags causing an error when trying to save maps.

> [Server] Fixed an issue with the initialization of the connection between Centreon Central and MAP that was causing issues with real time updates.

Reviewer: ```suggestion
- [Server] Fixed an issue with the initialization of the connection between Centreon Central and MAP that was causing issues with real time updates.
```

> - [ACL] Fixed an issue which caused user without creator role to be able to duplicate maps.
> - [Diagnostic] Fixed diagnostic when MAP is configured in HTTPS.
> - [Editor] Fixed an issue causing link labels not to be displayed in editor.
> - [Map list] Fixed an issue that could prevent the map list from being displayed.
> - [Server] Fixed an issue causing an error when trying to save maps.
> - [Server] Fixed an issue in connection between Centreon Central and MAP server initialization causing issues in real time updates.
> - [Server] Fixed concurrent access leading to inconsistencies in maps.

Reviewer: ```suggestion
- [ACL] Fixed an issue which caused some users to be able to duplicate maps when they shouldn't.
- [Diagnostic] Fixed an issue with the diagnostics tool that occurred when MAP was configured in HTTPS.
- [Editor] Fixed an issue causing link labels not to be displayed in editor mode.
- [Map list] Fixed an issue that could prevent the list of maps from being displayed.
- [Server] Fixed an issue with HTML tags causing an error when trying to save maps.
[Server] Fixed an issue with the initialization of the connection between Centreon Central and MAP that was causing issues with real time updates.
- [Server] Fixed an issue with concurrent accesses leading to inconsistencies in maps.
```

## Centreon MAP Legacy

### 23.04.18

Release date: `July 29, 2024`

<details>
  <summary>Bug fixes</summary>

> - [Diagnostic] Fixed diagnostic when MAP is configured in HTTPS.

Reviewer: ```suggestion
- [Diagnostic] Fixed the diagnostics performed when MAP was configured in HTTPS.
```

## Centreon MBI

### 23.04.10

Release date: `July 29, 2024`

<details open>
  <summary>Enhancements</summary>

> - [Selinux] Packaged selinux rules for centreon-map and centreon-mbi.

Reviewer: ```suggestion
- [Selinux] Packaged SELinux rules.
```
